pkg/gpu/stats.go

```diff
@@ -51,14 +54,15 @@ func (g *statsGenerator) getStats(nowKtime int64) *model.GPUStats {
 		}
 	}
 
-	g.configureNormalizationFactor()
+	normFactor := g.getNormalizationFactor()
 
 	stats := model.GPUStats{
-		PIDStats: make(map[uint32]model.PIDStats),
+		ProcessStats: make(map[uint32]model.PIDStats),
 	}
 
-	for pid, aggregator := range g.aggregators {
-		stats.PIDStats[pid] = aggregator.getStats()
+	for pid, aggr := range g.aggregators {
+		aggr.setGPUUtilizationNormalizationFactor(normFactor)
```

Comment:
for parity with the new `getNormalizationFactor`, let's rename it to `setNormalizationFactor`